packages/perseus/src/widgets/numeric-input/numeric-input.class.tsx

```diff
@@ -171,25 +170,12 @@ const propsTransform = function (
 ): RenderProps {
     // Omit the answers from the widget options since they are
     // not needed for rendering the widget.
-    const {answers: _, ...rendererProps} = {
-        ...widgetOptions,
-        answerForms: unionAnswerForms(
-            // Filter out the correct answers and map them to the answer forms
-            // so that we can generate the examples for the widget.
-            widgetOptions.answers
-                .filter((answer) => answer.status === "correct")
-                .map((answer) => {
-                    return (answer.answerForms || []).map((form) => {
-                        return {
-                            simplify: answer.simplify,
-                            name: form,
-                        };
-                    });
-                }),
-        ),
-    };
+    const {answers: _, fullAnswerForms = [], ...rendererProps} = widgetOptions;
 
-    return rendererProps;
+    return {
+        ...rendererProps,
+        answerForms: fullAnswerForms,
```

Comment:
Like I said, we might not keep it named `fullAnswerForms` so for now I'm just renaming it to `answerForms` so I don't have to update the widget code which I'd be fine doing once we have an established name.